src/wren/GlState.hpp

```diff
@@ -115,6 +115,7 @@ namespace wren {
     const char *version();
     const char *glslVersion();
 
+    int gpuMemory();
```

Comment:
I would add a comment here that explain that it's 0 for Intel cards, the currently available GPU memory for AMD cards and the total GPU memory for NVIDIA cards,